src/fides/api/schemas/connection_configuration/connection_secrets.py

```diff
@@ -13,10 +12,6 @@
 class ConnectionConfigSecretsSchema(BaseModel, abc.ABC):
     """Abstract Base Schema for updating Connection Configuration Secrets"""
 
-    url: Optional[str] = Field(
```

Comment:
Removing `url` from all DB connectors, we now require the user to provide the individual components.